jade/distribution.yaml

```diff
@@ -4313,6 +4313,15 @@ repositories:
       url: https://github.com/ros-controls/realtime_tools.git
       version: indigo-devel
     status: maintained
+  repository_name:
```

Comment:
Please choose a reasonable repository name, e.g. `rviz_ortho_view_controller`.

Afterwards the entry has to be moved up/down to stay in alphabetical order. Also when the repository name matches the only package name you so have to remove the `packages` list.